### ISSUE TITLE
installing via Carthage not working

### ISSUE THREAD

**Issue by ichina** [ReactorKit/ReactorKit#65]
`carthage update ReactorKit platform iOS`
result:
`*** Skipped building ReactorKit due to the error:
Dependency "ReactorKit" has no shared framework schemes`

**Comment by AidenJLee** [comment]
I have the same problem.

If you want to use it right now,

Clone the project and go to the terminal and type this.

 swift package generate-xcodeproj
 swift build
 

Then do the following actions

1. open & change xcode project setting (deployment target, os, release ..etc)
2. build
3. open a product folder 
4. copy & paste your project folder (Carthage/Build/iOS ...)


I do not know if this is a good way. but It is similar to the operation of Carthage.

I hope it helps.

**Comment by devxoul** [comment]
Hey guys, I've updated the prebuilt binary of 1.1.0. Please try again with it 😃

**Comment by ichina** [comment]
@devxoul the problem is still actual:
```Skipped building ReactorKit due to the error:
Dependency "ReactorKit" has no shared framework schemes for any of the platforms: iOS

If you believe this to be an error, please file an issue with the maintainers at https://github.com/ReactorKit/ReactorKit/issues/new```

**Comment by devxoul** [comment]
@ichina Could you please try after clearing caches?

```console
$ rm -rf ~/Library/Caches/carthage
$ carthage update
```

**Comment by ripplek** [comment]
Please support Carthage as soon as possible

```
Skipped building ReactorKit due to the error:
Dependency "ReactorKit" has no shared framework schemes
```

**Comment by RossSong** [comment]
Please support Carthage as soon as possible
----
*** Downloading ReactorKit.framework binary at "1.1.0"
***  Skipped installing ReactorKit.framework binary due to the error:
	"Incompatible Swift version - framework was built with 4.1 (swiftlang-902.0.48 clang-902.0.37.1) and the local version is 4.1.2 (swiftlang-902.0.54 clang-902.0.39.2)."

    Falling back to building from the source
*** Skipped building ReactorKit due to the error:
Dependency "ReactorKit" has no shared framework schemes

**Comment by yhkaplan** [comment]
Hey anyone still encountering this problem. Please refer to this thread: https://github.com/ReactorKit/ReactorKit/issues/81

TLDR: Use `(cd Carthage/Checkouts/ReactorKit && swift package generate-xcodeproj)`, then `carthage build` if it isn't working
